pontis-ui/utils/encodeUrl.ts

```diff
@@ -0,0 +1,19 @@
+import { AssetKeyT } from "../src/hooks/oracle";
+
+/**
+ *
+ * @param {AssetKeyT} assetKey
+ * @return {string}
+ */
+export function assetKeyToUrl(assetKey: AssetKeyT): string {
+  return encodeURIComponent(assetKey.replace("/", "-"));
+}
+
+/**
+ *
+ * @param {string} url
+ * @return {AssetKeyT}
+ */
+export function urlToAssetKey(url: string): AssetKeyT {
+  return decodeURIComponent(url.replace("-", "/"));
```

Comment:
It will correctly convert the future because `replace` only replaces the first occurences (unlike `replaceAll`). Yeah, we should think more about testing.